From 95bab2728cc0597adadd6eb673b20d03ce67fdbe Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 5 Feb 2021 13:53:12 -0700
Subject: [PATCH 05/12] [Q_asb_2021-02] Bump Security String to 2021-02-05

Implemented:
============
CVE:            References:  Type:  Severity:  Updated AOSP versions:
CVE-2021-0302   A-155287782  EoP    High       8.1, 9, 10
CVE-2021-0305   A-154015447  EoP    High       8.1, 9, 10
CVE-2021-0314   A-171221302  EoP    High       8.1, 9, 10, 11
CVE-2021-0325   A-174238784  RCE    High       10, 11
                             RCE    Critical   8.1, 9
CVE-2021-0326   A-172937525  RCE    Critical   8.1, 9, 10, 11
CVE-2021-0327   A-172935267  EoP    High       8.1, 9, 10, 11
CVE-2021-0328   A-172670415  EoP    High       8.1, 9, 10, 11
CVE-2021-0329   A-171400004  EoP    High       8.1, 9, 10, 11
CVE-2021-0330   A-170732441  EoP    High       9, 10, 11
CVE-2021-0331   A-170731783  EoP    High       8.1, 9, 10, 11
CVE-2021-0332   A-169256435  EoP    High       10, 11
CVE-2021-0333   A-168504491  EoP    High       8.1, 9, 10, 11
CVE-2021-0334   A-163358811  EoP    High       8.1, 9, 10, 11
CVE-2021-0336   A-158219161  EoP    High       8.1, 9, 10, 11
CVE-2021-0337   A-157474195  EoP    High       8.1, 9, 10, 11
CVE-2021-0338   A-156260178  DoS    High       10, 11
CVE-2021-0339   A-145728687  EoP    High       8.1, 9, 10
CVE-2021-0340   A-134155286  EoP    High       10
CVE-2021-0341   A-171980069  ID     High       8.1, 9, 10, 11

Not Implemented:
================
None

Not Applicable (platform source):
=================================
CVE:            References:  Type:  Severity:  Updated AOSP versions:
CVE-2021-0335   A-160346309  ID     High       11

Change-Id: I02618c0562b4c58601c18ecfbf411ec115c6f89e
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 3cc9c74c9..8fdcef023 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -256,7 +256,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2021-01-05
+      PLATFORM_SECURITY_PATCH := 2021-02-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.34.1

